src/main/resources/db/changelog/changes/15-create-orders-order-items-table.yaml

```diff
@@ -0,0 +1,32 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-orders-order-items-table
```

Comment:
why is it done here, orders were not working before that change I assume? But I guess we can leave it here